samples/python/multinomial_causal_lm/README.md

```diff
@@ -19,7 +19,10 @@ optimum-cli export openvino --trust-remote-code --model TinyLlama/TinyLlama-1.1B
 
 Install [deployment-requirements.txt](../../deployment-requirements.txt) via `pip install -r ../../deployment-requirements.txt` and then, run a sample:
 
-`python multinomial_causal_lm.py TinyLlama-1.1B-Chat-v1.0 "Why is the Sun yellow?"`
+#### Simple streaming
+`python multinomial_causal_lm.py -m TinyLlama-1.1B-Chat-v1.0 -p "Why is the Sun yellow?" -tl 1`
+#### Chunk streaming
+`python3 multinomial_causal_lm.py -m TinyLlama-1.1B-Chat-v1.0 -p "Why is the Sun yellow?" -tl 10`
```

Comment:
Since I requested to remove `t1` cmd arg, this section should be removed.

There's a section below about streaming. Please extend it to cover chunking as well.